lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -580,10 +584,10 @@ llvm::Error Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     }
 
     // Caffe2 stores the transposed W matrix. In here we first coerce W to a 2D
-    // matrix size if necessay and then transpose it back.
+    // matrix size if necessary and then transpose it back.
     Tensor tmp;
+    auto wDims = flattenCdr(w->dims(), axis_w);
```

Comment:
qq: why do we flatten no matter what (check `w->dims().size() > 2` seems reasonable).

Reply:
This doesn't flatten, it just finds the dims if we do flatten. I put this here so that it's accessible below.